ach/src/main/java/com/adyen/checkout/ach/internal/ui/StoredACHDirectDebitDelegate.kt

```diff
@@ -108,6 +118,9 @@ internal class StoredACHDirectDebitDelegate(
         adyenLog(AdyenLogLevel.ERROR) { "updateInputData should not be called in StoredACHDirectDebitDelegate" }
     }
 
+    @Suppress("ForbiddenComment")
+    // TODO: Here we only call this method on initialization. The checkoutAttemptId will only be available if it is
+    //  passed by drop-in. This should be fixed as part of state refactoring.
```

Comment:
Let's discuss this as this probably is a problem for all stored payment methods and instant payment methods